src/clientdlg.cpp

```diff
@@ -1132,6 +1156,9 @@ OnTimerStatus();
 
     // clear mixer board (remove all faders)
     MainMixerBoard->HideAll();
+
+    // clear recording state
+    MainMixerBoard->SetServerRecordingState( "" );
```

Comment:
There are a lot of places where a space is missing before the (.

Reply:
Hopefully all fixed now (in files touched by this patch, at least...).